tests/sapi.rs

```diff
@@ -102,3 +117,91 @@ pub fn hello_world(name: String) -> String {
 pub fn module(module: ModuleBuilder) -> ModuleBuilder {
     module.function(wrap_function!(hello_world))
 }
+
+#[test]
```

Comment:
```suggestion
#[test]
#[cfg(php_zts)]
```

Reply:
Ah, you meant skipping it altogether. Makes sense. Thanks for the guidance!